Align Language and Fix Grammar/Spelling Across Edudoc Files

sarahstrong314 opened the pull request:
### Consistency with WCA Website and Documents

- **Terminology**
	- cube cover → puzzle cover
	- puzzle drop-off station → puzzle drop-off table
	- competing area → solving stations
	- competitor waiting area → waiting area
	- scrambling station → scrambling table
	- stationary judges → seated judges
	- non-Delegate organizer → Community organizer
	- registration terms and conditions → registration requirements
	- winner's ceremony → awards ceremony
	- Round 1 → First Round (when referring to a specific first round)
	- round 1 → the first round (when referring to a general first round)
	- +2 → +2 penalty
	- US → USA
	- United States → USA
- **Capitalization**
 	- Delegate Report
 	- Regional Delegate
 	- Regional Organization
 	- Regulations
 	- WCA Team
 	- WCA Committee
 	- Community Member
 	- Extra Registration Requirements

### Consistency with WCA Regulations

- **Terminology**
	- scorecard → score sheet
	- data entry → score-taking (noun or adjective)
	- data entry person → score taker
	- extra → extra attempt
	- provisional extra → provisional extra attempt
	- organizational team → organization team
	- organizer team → organization team
	- organizational staff → organizers
	- member of the organization team → organizer
	- issue/issuing extra attempts → grant/granting extra attempts
	- ex. → e.g.
- **Style**
	- write all event names in full
	- expand contractions
	- avoid sentences that end in "etc."
	- no auto-capitalizing after "e.g."
	- no auto-capitalizing after colons (except "Note:")
	- use curved quotation marks (' → ’, " → “/”, except in HTML code)
	- put punctuation marks outside of quotation marks

### General Consistency

- **Terminology**
	- first-timer → first-time competitor
	- new competitor → first-time competitor
	- new-comer/newcomer → first-time competitor 
 		- This is an important distinction since being a newcomer doesn't imply it's their first time, just that they started competing recently.
	- e-mail → email
	- gmail/GMail → Gmail
	- insurance fee → liability insurance coverage
	- Delegates of the competition → listed Delegates
- **Style**
	- use bold for emphasis instead of upper casing (except for speech)
	- remove colons at the end of headers
	- remove "WCA" from `h1` tags
	- title casing in `h1` and `h2` tags
	- sentence casing in `h3` tags (except in Delegate Handbook)
	- remove `h4` tags that aren't under a `h3` tag
	- use `/` only with interchangeable terms (e.g. Committee/Team or parents/guardians)
	- remove trailing whitespace at the end of lines
- **Abbreviations and Acronyms**
	- write all Committee/Team names in full (only the first time they appear in a section)
	- RO → Regional Organization
	- 3x3 → 3x3x3 Cube (full event name)
	- OH → 3x3x3 One-Handed (full event name)
	- 3BLD → 3x3x3 Blindfolded (full event name)
	- CC → Carbon Copy (only the first time it appears in a section)
	- BCC → Blind Carbon Copy (only the first time it appears in a section)
	- UTC → Coordinated Universal Time (only the first time it appears in a section)
- **Alignment with American English**
	- organisation → organization
	- organiser → organizer
	- public transport → public transportation
	- insurance cover → insurance coverage
	- extension lead → extension cord
	- query → question
	- towards → toward
	- amongst → among
	- offence → offense
	- favourite → favorite
- **Inclusivity**
	- reword "whitelist"
	- ~dummy → bogus (e.g. dummy data → bogus data)~ (**changed to _mock_ data in second commit**)
	- weird → unusual
	- parents → parents/guardians

### General Corrections

- **Grammar Fixes**
	- waive registration → waive registration fee
	- press → click (when referring to a mouse click)
	- likelihood → possibility (when referring to possibility)
	- write ordinals in full (e.g. 1st → first)
	- ensure a complete, independent clause precedes every colon that introduces a list
	- split run-on sentences
	- fix comma splices
	- fix incorrect prepositions
	- fix dangling modifiers
	- add missing commas
	- add Oxford commas
- **Spelling Fixes**
	- Scramble Matcher → Scramble**s** Matcher
	- flier → flyer
	- vice-versa → vice versa
	- check-in (noun or adjective)
	- check in (verb)
	- setup (noun or adjective)
	- set up (verb)
	- on-the-spot (adjective)
	- on the spot (verb)
	- world record (noun)
	- world-record (adjective)
- **Terminology Corrections** 
	- revenue → net profit (Important: the image still needs to be updated to reflect this)

### Readability and Translatability Improvements

- **Simplified Terminology**
	- begin → start
	- mitigate → reduce
	- take into account → consider
	- keeping in mind → considering
	- keep in mind → remember
	- feel free → you may
	- drastic/dramatic → considerable (when appropriate)
	- major → significant
	- communicative → in communication
	- dealing with → interacting with _or_ managing (when appropriate)
	- in lieu of → instead of
	- take advantage of → use (when referring to funding)
	- dictates → outlines/indicates
	- in attendance → present
	- forgo → exclude
	- highlight → emphasize
	- beneficial → useful/helpful
	- track record → history
	- desire → wish
	- opts → choose (except when "opt" is used as a technical term)
	- via → per (when referring to Regulations)
	- reword "nefarious actors"
- **Consistent Terminology**
	- pass cutoff → meet cutoff
	- as opposed to → instead of
	- make sure → ensure
	- be sure → ensure
	- unsure → uncertain
	- not sure → uncertain
	- not clear → unclear
	- note that → remember that
	- individual (noun) → person
	- to/in CC/BCC → to the CC/BCC field 
	- competition organizers → organizers
	- state of the puzzle → puzzle state
	- organizers of the competition → organizers
	- rest of the organization team → co-organizers
	- Team or Committee → Committee/Team (**updated in third commit**)
	- Committee or Team → Committee/Team (**updated in third commit**)
- **Avoiding Idioms**
	- reword "100 competitions under your belt"
	- reword "which hat you are wearing"
	- reword "learn the ropes"
	- reword "slip through the cracks"
	- keep an eye out for → watch out for
	- keep on top of → stay up to date with
	- stay on top of → keep up with
	- on track → on schedule
	- last-minute → very end _or_ near the end
- **Simplified Grammar**
	- as well as → and (when appropriate)
	- on top of that → also
	- in order to → to
	- whether or not → whether
	- have to → need to
	- attempt to → try to
	- avoid semicolons and dashes (N.B.: hyphens are not dashes)
	- use exclamation point sparingly
	- use "very" sparingly
	- use plural as default instead of appending "(s)" to a noun
	- avoid using "not" when there is an appropriate opposite word
- **Grammar Improvements**
	- which → that (when appropriate)
	- in cases where → when
	- in case → if (when appropriate)
	- in the case that → if/when
	- if → when (when appropriate)
	- if → whether (when appropriate)
	- have to → need to
	- need to → must (when appropriate)
	- can/could → may (when referring to permission or having a right)
	- can/could → might (when referring to possibility)
	- may → can (when referring to abilty, not permission)
	- may → could (when referring to a suggestion)
	- may → might (when referring to possibility)
	- is able to → can (when appropriate)
	- may be able to → could
	- it is recommended that → you should
	- a handful of → several
	- a number of → many/several
	- a percentage of → many/several
	- normal → regular (when it means "standard" or "as expected")
	- normal → usual (when it means "common")
	- all → every (when there is emphasis on each individual)
	- consistent tenses in tables
	- consistent casing for bullet points
	- reword phrases in passive tense to be in active tense (when appropriate)
- **Clarity Improvements**
	- region → country (when referring to representation)
	- accommodations → lodging (when referring to lodging)
	- parking space → parking spots
	- avoid podium as a verb
	- certain time → specific time

### Markdown and HTML

- **Markdown**
	- add `mailto:` to every email address
	- add missing links to other edudocs
	- correct spacing around `**`
	- use `-` for bullet points instead of `*`
	- align `|` delimiters in tables
- **HTML**
	- remove unnecessary `\` at the end of lines
	- close all `<tr>` tags
	- ~use tabs instead of spaces to indent nested `<tr>` and `<td>` tags~ (**replaced with two spaces in second commit**)

dmint789 commented:
Likewise, the right term for the competing area is competition area

dmint789 commented:
>put punctuation marks outside of quotation marks

Aren't they supposed to be inside? Also, do we need to replace uses of wca staff with wca volunteers?

dmint789 commented:
For acronyms, did you also include the acronym in parentheses wherever you used it for the first time? E.g. Carbon Copy (CC). That's how we normally do it.

dmint789 commented:
Instead of bogus data I'd use "mock data"

dmint789 commented:
Do you need help with updating the images?

dmint789 commented:
Also, please undo all of the changes that replace spaces with tabs. We don't use tabs in these types of projects anymore (code, markdown, etc.).

dmint789 commented:
@roxanajohnson88 my bad, did that by accident

dmint789 reviewed (changes requested):
Left comments

dmint789 commented on edudoc/competition-templates/competition-templates.md:
```diff
@@ -1,29 +1,30 @@
 
 ::::: {.box .important}
-DO NOT use any templates before reading this entire document!
+Please read this entire document before using any templates!
```
I really wanted this line to stand out, hence the all caps

sarahstrong314 commented:
I've replaced "puzzle drop-off table" with "drop-off table", one instance of "solving stations" with "competition area" (since it was originally "competing area"), two instances of "bogus" with "mock", and tabs with two spaces (which most documents originally had). I also added caps back to the first sentence in the Competition Templates document.

sarahstrong314 commented:
> > put punctuation marks outside of quotation marks
> Aren't they supposed to be inside? Also, do we need to replace uses of wca staff with wca volunteers?

Yes according to most style guides. However, the Regulations do not follow the common style. The Regulations also don't put commas after "e.g." and "i.e." although most style guides recommend this.

> Do you need help with updating the images?
Yes please!

dmint789 commented:
@sarahstrong314 can you please give me a list of images that need updating and what text should be changed to what

sarahstrong314 commented:
> @sarahstrong314 can you please give me a list of images that need updating and what text should be changed to what


`edudoc/organizer-guidelines/images/budget.png`

"Final revenue" should be "Net profit".

![](https://raw.githubusercontent.com/thewca/wca-documents/refs/heads/master/edudoc/organizer-guidelines/images/budget.png)

sarahstrong314 commented:
I've uploaded the new `budget.png` images provided by @dmint789.

dmint789 reviewed (approved):
LGTM!
